examples/java-agent/README.md

```diff
@@ -0,0 +1,14 @@
+This SparkJava-based example builds a container image that includes the [Stackdriver Debugger Java Agent](https://cloud.google.com/debugger/docs/).
+This project assumes the resulting image will be run inside Google CLoud Platform. To run outside of
+GCP, follow the Stackdrive Debugger instructions to [create and download service account
+credentials](https://cloud.google.com/debugger/docs/setup/java#local). 
```

Comment:
This change definitely seems better, but after hearing what @saturnism said, I think we should get rid of anything that can make people even remotely think about putting a credential into a container. Such idea seems really dangerous. The Stackdriver debugger instructions are under "Local and Elsewhere", and I'm not sure if the "Elsewhere" part includes running containers on prod outside GCP, but now I'd like to avoid making the practice of embedding a key in a container look like a viable solution, even remotely. Maybe we can append at the end of the paragraph "and inject the credentials in the secure way" or "never embed a key into the container, since it becomes easy to make it accessible from others" or "the credentials are secret that should be kept secure".

Reply:
I'll remove that text, and will submit feedback on the Stackdriver Debugger doc page.